iohk/NixOps.hs

```diff
@@ -820,29 +519,33 @@ cmd   Options{..} bin args = do
 cmd'  Options{..} bin args = do
   when (toBool oVerbose) $ logCmd bin args
   Turtle.procStrict bin args empty
+cmd'' Options{..} command  = do
+  when (toBool oVerbose) $ logCmd command []
+  Turtle.shellStrictWithErr command empty
 incmd Options{..} bin args = do
   when (toBool oVerbose) $ logCmd bin args
   inprocs bin args empty
 
 
 -- * Invoking nixops
 --
-iohkNixopsPath :: FilePath -> FilePath
-iohkNixopsPath defaultNix =
-  let storePath  = Sys.unsafePerformIO $ inprocs "nix-build" ["-A", "nixops", format fp defaultNix] $
+iohkNixopsPath :: FilePath
+iohkNixopsPath =
+  let defaultNix = "default.nix"
+      storePath  = Sys.unsafePerformIO $ inprocs "nix-build" ["-A", "nixops", format fp defaultNix] $
                    (trace (T.unpack $ format ("INFO: using "%fp%" expression for its definition of 'nixops'") defaultNix) empty)
-      nixopsPath = Path.fromText $ T.strip storePath <> "/bin/nixops"
-  in trace (T.unpack $ format ("INFO: nixops is "%fp) nixopsPath) nixopsPath
+      opsPath    = Path.fromText $ T.strip storePath <> "/bin/nixops"
```

Comment:
We have `nixops` and `iohk-ops`, so `ops` is an overloaded term, why make the name binding ambigious?

Reply:
This is a local variable, and the meaning is very clear, IMO -- its usage is really spread across three lines.   But ok.